ArduPlane/tailsitter.cpp

```diff
@@ -805,12 +805,13 @@ void Tailsitter::write_log()
         return;
     }
 
-    struct log_tailsitter pkt = {
+    struct log_TSIT pkt = {
         LOG_PACKET_HEADER_INIT(LOG_TSIT_MSG),
         time_us             : AP_HAL::micros64(),
         throttle_scaler     : log_data.throttle_scaler,
         speed_scaler        : log_data.speed_scaler,
         min_throttle        : log_data.min_throttle,
+        transition_state    : uint8_t(transition->transition_state),
```

Comment:
Tailsitter still logs in `QTUN`, do we need it here too?

Reply:
> Tailsitter still logs in `QTUN`, do we need it here too?

Different enumerations, so I needed a different spot.  I was wondering about creating an `SLT` message...

Reply:
But QTUN still logs with two enums? So this does not solve that problem? We have them manually documented:

```
// @Field: Trn: Transition state: 0-AirspeedWait,1-Timer,2-Done / TailSitter: 0-FW Wait,1-VTOL Wait,2-Done
```

Reply:
Thus my musings about an SLT message...